### PR TITLE
reflecting support for MATLAB builds

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

**Comment by mw-hrastega** [comment]
Hi Sam and Mark,

Any comments on the changes made to the overview?

Thanks,
Houman

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+The `MATLAB.gitlab-ci.yml` [template](https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/MATLAB.gitlab-ci.yml) provides you with jobs that show how to run a MATLAB build, as well as MATLAB tests, scripts, functions, and statements. To run MATLAB code and Simulink models based on this template, you must use the Docker executor to run MATLAB within a container. The [MATLAB Container on Docker Hub](https://www.mathworks.com/help/cloudcenter/ug/matlab-container-on-docker-hub.html) lets you run your build using MATLAB R2020b or a later release. If your build requires additional toolboxes, use a custom MATLAB container instead. For more information on how to create and use a custom MATLAB container, see [Create a Custom MATLAB Container](https://www.mathworks.com/help/cloudcenter/ug/create-a-custom-matlab-container.html).
```
Comment (sameagen-MW) [inline README.md:4]: Do we want to present the jobs in the order that they appear in the template?

Comment (sameagen-MW) [inline README.md]: Do we want to list the jobs in the same order they appear in the template?

File: README.md [hunk 2]
```diff
@@ -84,6 +85,15 @@ artifacts:
       - "./artifacts"
 ```
 
+### Run MATLAB Build
+Use the [MATLAB build tool](https://www.mathworks.com/help/matlab/matlab_prog/overview-of-matlab-build-tool.html) to run the build plan returned by a file named `buildfile.m` in the root of your repository. Run the default tasks in your plan as well as all the tasks on which they depend.
```
Comment (sameagen-MW) [inline README.md]: This doesn't quite match the comment that we used on the template submission, but I like both. I just want to make sure that it's not an issue that they aren't identical.

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md]: I think it's better if the comment align. Updated the text.

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md]: Good suggestion. Updated.

**Review by sameagen-MW — APPROVED** [review]

**Pull request merged**